controller/engine_controller.go

```diff
@@ -67,7 +69,8 @@ type EngineController struct {
 	eStoreSynced cache.InformerSynced
 	pStoreSynced cache.InformerSynced
 
-	queue workqueue.RateLimitingInterface
+	backoff *flowcontrol.Backoff
+	queue   workqueue.RateLimitingInterface
```

Comment:
This change and one below about queue are unnecessary. Can switch the line position fix it?

Reply:
By switching the line, do you mean switching the order that the `queue` and `backoff` properties are defined right now? That doesn't prevent this change, since `go fmt` still attempts to get the `type` here and the value below to line up, and `git` still treats it as a removal of the original `queue` line.

Reply:
Oh right...